.eslintrc

```diff
@@ -1,39 +1,45 @@
 {
-    "env": {
-      "browser": true,
-      "commonjs": true,
-      "es6": true,
-      "node": true,
-      "jest": true
-    },
-    "plugins": [
-      "react",
-      "react-hooks",
-      "jest",
-      "better-styled-components"
-    ],
-    "extends": ["eslint:recommended", "plugin:react/recommended", "plugin:prettier/recommended"],
-    "parserOptions": {
-      "sourceType": "module",
-      "ecmaVersion": 12
-    },
-    "settings": {
-      "react": {
-        "version": "detect"
-      }
-    },
-    "rules": {
-      "linebreak-style": ["warn", "unix"],
-      "no-unused-vars" : "warn",
-      "react-hooks/rules-of-hooks": "error",
-      "react-hooks/exhaustive-deps": "warn",
-      "react/react-in-jsx-scope": "off",
-      "react/prop-types": "off",
-      "jest/no-disabled-tests": "warn",
-      "jest/no-focused-tests": "error",
-      "jest/no-identical-title": "error",
-      "jest/prefer-to-have-length": "warn",
-      "jest/valid-expect": "error",
-      "better-styled-components/sort-declarations-alphabetically": "warn"
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es6": true,
+    "node": true,
+    "jest": true
+  },
+  "plugins": [
+    "react",
+    "react-hooks",
+    "jest",
+    "better-styled-components",
+    "cypress"
+  ],
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:prettier/recommended",
+    "plugin:cypress/recommended"
+  ],
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaVersion": 12
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
     }
-  }
\ No newline at end of file
+  },
+  "rules": {
+    "linebreak-style": ["warn", "unix"],
+    "no-unused-vars": "warn",
```

Comment:
Should we not consider this as 'error' or 2?

Reply:
Sorry I didn't get you here.